### PR TITLE
FIX #48 - Rename mysql -> mysql-k8s

### LINKED ISSUE #48: Change the name everywhere
to: application-k8s

### PR DESCRIPTION
**Pull request by Abuelodelanada**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,23 +11,23 @@ The [MySQL](https://www.mysql.com/) operator provides an open-source relational
 ## Deployment instructions
 
 
-Create a Juju model for your operators, say "mysql"
+Create a Juju model for your operators, say "mysql-k8s"
 
 ```bash
-$ juju add-model mysql
+$ juju add-model mysql-k8s
```
Comment (sed-i) [inline README.md:17]: I think this needs to move into `CONTRIBUTING.md` or `INTEGRATION.md`. @jlounder?

File: README.md [hunk 2]
```diff
@@ -11,23 +11,23 @@ The [MySQL](https://www.mysql.com/) operator provides an open-source relational
...
 ```
 
 Deploy a single unit of MySQL using its default configuration
 
 ```bash
-$ juju deploy ./mysql.charm --resource mysql-image=ubuntu/mysql:latest
+$ juju deploy ./mysql-k8s.charm --resource mysql-image=ubuntu/mysql:latest
```
Comment (sed-i) [inline README.md:23]: Shouldn't this be written as if deploying from charmhub?

File: README.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ This charm implements the following configs:
 And you can use it, like this:
 
 ```bash
-$  juju deploy ./mysql.charm --config MYSQL_ROOT_PASSWORD=SuperSecretPassword --config MYSQL_USER=JohnDoe --config MYSQL_PASSWORD=SuperSecretUserPassword --config MYSQL_DATABASE=default_database --resource mysql-image=ubuntu/mysql:lates
+$  juju deploy ./mysql-k8s.charm --config MYSQL_ROOT_PASSWORD=SuperSecretPassword --config MYSQL_USER=JohnDoe --config MYSQL_PASSWORD=SuperSecretUserPassword --config MYSQL_DATABASE=default_database --resource mysql-image=ubuntu/mysql:lates
```
Comment (sed-i) [inline README.md:57]: Shouldn't this be written as if deploying from charmhub?

File: README.md (hunk 2 above)
Comment (Abuelodelanada) [reply README.md:23]: I was thinking in changing this once the charm is uploaded to charmhub

File: README.md (hunk 3 above)
Comment (Abuelodelanada) [reply README.md:57]: I was thinking in changing this once the charm is uploaded to charmhub

**Review by sed-i — APPROVED** [review]

**Review by dstathis — APPROVED** [review]

**Pull request merged**